install_script.sh.template

```diff
@@ -651,7 +651,7 @@ fi
 config_file_fips=$etcdirfips/datadog-fips-proxy.cfg
 system_probe_config_file=$etcdir/system-probe.yaml
 security_agent_config_file=$etcdir/security-agent.yaml
-environment_file=$etcdir/environment
+environment_file=/etc/environment
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ Since this variable is only used in one place, why not just pass the `/etc/environment` value directly?

Reply:
okay, done in af52838